### PR TITLE
pkp/pkp-lib#2338 enable context specific usage stats display options

### PR DESCRIPTION
**Pull request by bozana**
s. https://github.com/pkp/pkp-lib/issues/2338

### CODE REVIEW

File: plugins/generic/usageStats/locale/en_US/locale.xml [hunk 1]
```diff
@@ -34,6 +34,8 @@
 	<message key="plugins.generic.usageStats.settings.compressArchives.description">Activate this option to compress archived log files using gzip tool (you will have to configure the gzip setting inside config.inc.php). If you have a high traffic site it's a good idea to enable this, so you can save some extra disk space.</message>
 	<message key="plugins.generic.usageStats.settings.compressArchives">Compress archives</message>
 	<message key="plugins.generic.usageStats.settings.statsDisplayOptions">Statistics display options</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.contextWide">The following display settings apply only to the current context:</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.siteWide">The following display settings will be used as default i.e. for each context that does not overwrite them:</message>
```
Comment (NateWr) [inline plugins/generic/usageStats/locale/en_US/locale.xml]: "Each journal can override these settings from the journal's plugins page."

File: plugins/generic/usageStats/locale/en_US/locale.xml [hunk 2]
```diff
@@ -34,6 +34,8 @@
 	<message key="plugins.generic.usageStats.settings.compressArchives.description">Activate this option to compress archived log files using gzip tool (you will have to configure the gzip setting inside config.inc.php). If you have a high traffic site it's a good idea to enable this, so you can save some extra disk space.</message>
 	<message key="plugins.generic.usageStats.settings.compressArchives">Compress archives</message>
 	<message key="plugins.generic.usageStats.settings.statsDisplayOptions">Statistics display options</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.contextWide">The following display settings apply only to the current context:</message>
```
Comment (NateWr) [inline plugins/generic/usageStats/locale/en_US/locale.xml]: "These settings will only be applied to usage statistics on {$journalName}."

File: plugins/generic/usageStats/templates/usageStatsSettingsForm.tpl [hunk 3]
```diff
@@ -46,6 +46,14 @@
...
 		{/fbvFormSection}
 	{/fbvFormArea}
+
+	<h3>
+	{if $currentContext}
+		{translate key="plugins.generic.usageStats.settings.statsDisplayOptions.contextWide"}
+	{else}
+		{translate key="plugins.generic.usageStats.settings.statsDisplayOptions.siteWide"}
+	{/if}
+	</h3>
```
Comment (NateWr) [inline plugins/generic/usageStats/templates/usageStatsSettingsForm.tpl]: Move the note into the `{fbvFormArea}` and use a `<p>` or `<div>` element instead, ie:

```
{fbvFormArea id="usageStatsDisplayOptions" title="plugins.generic.usageStats.settings.statsDisplayOptions"}
	<p>
		{if $currentContext}
			{translate key="plugins.generic.usageStats.settings.statsDisplayOptions.contextWide"}
		{else}
			{translate key="plugins.generic.usageStats.settings.statsDisplayOptions.siteWide"}
		{/if}
	</p>
	{fbvFormSection for="displayStatistics" list=true}
```

File: plugins/generic/usageStats/locale/en_US/locale.xml (hunk 1 above)
Comment (bozana) [inline plugins/generic/usageStats/locale/en_US/locale.xml]: Do you maybe have a suggestion, how to make this text OJS and OMP neutral i.e. without 'journal' or 'press'? Can we use context?

Comment (NateWr) [inline plugins/generic/usageStats/locale/en_US/locale.xml]: I don't think OMP has multi-context setup at the moment, so I think it's ok to use "journal".

I don't think anyone understands "context" except OJS/OMP developers. :) If necessary, I guess you'll need two strings and to check what application is running.

Comment (bozana) [inline plugins/generic/usageStats/locale/en_US/locale.xml]: I separated the keys into the applications...

**Pull request merged**